gradle-plugin/doc/src/main/asciidoc/inc/_groovy-configuration.adoc

```diff
@@ -207,6 +207,24 @@ This option is only relevant for the `{goal-prefix}:build` goal.
 | *logStdout*
 | Log to stdout regardless if log files are configured or not. Default is `false`.
 | `jkube.docker.logStdout`
+
+| *logFollow*
+| Get follow logs for your application inside Kubernetes.
+
+  Defaults to `true`.
+| `jkube.log.follow`
+
+| *logContainer*
+| Get logs of some specific container inside your application Deployment.
+
+  Defaults to `null`.
+| `jkube.log.container`
+
+| *logPod*
+| Get logs of some specific pod inside your application Deployment.
+
+  Defaults to `null`.
+| `jkube.log.pod`
```

Comment:
We need to discuss about the order of the table entries. It's now partially sorted alphabetically. Remind me if I forget about this.

Reply:
If it's already partially sorted I can go ahead and sort this.

Reply:
Do as you wish. But we'll need to see this once we review the final documentation.

Reply:
okay. I'll sort this here and keep sorting order in mind while adding new entries.